### PR TITLE
fix: full bleed adjustments

### PR DESCRIPTION
**Pull request by stephiescastle**
### Checklist

- [x] Include a description of your pull request and instructions for the reviewer to verify your work.
- [x] Link to the issue if this PR is issue-specific.
- [x] Create/update the corresponding story if this includes a UI component.
- [ ] Create/update documentation. If not included, tell us why.
- [x] List the environments / browsers in which you tested your changes.
- [x] Tests, linting, or other required checks are passing.
- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../releases); good ones make that easier to scan.
  - PRs will be broadly categorized in the change log, but for even easier scanning, consider prefixing with a component name or other useful categorization, e.g., "BaseButton: fix layout bug", or "Storybook: Update dependencies".
- [x] PR has been tagged with a [SemVer](https://semver.org/) label and a [general category label](https://github.com/nasa-jpl/explorer-1/blob/52994b671411f55961d7beb8851d4580ac3f434f/.github/release-drafter.config.yml#L21-L39), or skip-changelog.
  - These tags are used to do the aforementioned broad categorization in the change log and determine what the next release's version number should be.
  - [Release Drafter](https://github.com/marketplace/actions/release-drafter) will attempt to do the category labeling for you! Please double-check its work.

## Description

Addresses feedback:
- https://github.com/nasa-jpl/www/issues/601#issue-2546642664

Changes:
- EDU NavDesktop will now expand to full-width
- Removed gray extenders in `HeroMedia` and `BlockImageFullBleed` site-wide
- Modified BlockCardGrid to only expand to four columns if there are at least four cards. I also changed the breakpoint for when this expands, as it's previous breakpoint didn't make much sense (its container wasn't changing size, just the viewport).

## Instructions to test

1. `make vue-storybook`
2. Full width EDU Nav: http://localhost:6006/?path=/story/navigation-headers-edu-navdesktopedu--base-story&globals=theme:ThemeEdu
3. Remove gray extenders:
    - View http://localhost:6006/?path=/story/components-blocks-blockimage--full-bleed
    - View http://localhost:6006/?path=/story/components-heroes-media-only--base-story
4. BlockCardGrid
    - Only three cards: http://localhost:6006/?path=/story/components-blocks-blockcardgrid--three-cards&globals=theme:ThemeEdu
    - Context-aware breakpoint for 4 columns:
      - View in Streamfield: http://localhost:6006/?path=/story/components-blocks-blockstreamfield--base-story (scroll down to the bottom)
      - View in Robotics template: http://localhost:6006/?path=/story/templates-www-pageroboticsdetail--template

### Operating System

- [x] macOS
- [ ] iOS
- [ ] iPadOS
- [ ] Windows

### Browser

- [ ] Chrome
- [ ] Firefox ESR
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: packages/vue/src/components/NavDesktopEdu/NavDesktopEdu.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
     }"
   >
     <!-- navbar -->
-    <div class="header-bg z-10 max-w-screen-3xl absolute inset-0 mx-auto"></div>
+    <div class="header-bg z-10 absolute inset-0 mx-auto"></div>
```
Comment (Scotchester) [inline packages/vue/src/components/NavDesktopEdu/NavDesktopEdu.vue:15]: 1. I think this class also needs to be removed from the beta banner.
2. Any reason to not also remove it from the WWW desktop nav?

Comment (stephiescastle) [reply packages/vue/src/components/NavDesktopEdu/NavDesktopEdu.vue:15]: 1. Yes, plan to do that in www PR that also updates the explorer-1 version
2. Yes, due to the nature of the nav sometimes being transparent with a gradient, and sometimes solid white.

**Review by Scotchester — APPROVED** [review]

File: packages/vue/src/components/BlockCardGrid/BlockCardGrid.stories.js [hunk 2]
```diff
@@ -92,3 +92,6 @@ export const BaseStory = {
   name: 'BlockCardGrid',
   args: { cards: BlockCardGridData }
 }
+export const ThreeCards = {
+  args: { cards: BlockCardGridData.slice(0, 3) }
+}
```
[the remark this answers is not in the document]
Comment (stephiescastle) [reply packages/vue/src/components/BlockCardGrid/BlockCardGrid.stories.js:97]: Yes, it's a new story to cover the use-case when there are only 3 cards

[the remark this answers is not in the document]
Comment (stephiescastle) [reply packages/vue/src/components/BlockCardGrid/BlockCardGrid.stories.js:97]: Here it is on chromatic: https://668c47cbeb95392cd79c3c0d-xiuixuscxz.chromatic.com/?path=/story/components-blocks-blockcardgrid--three-cards

[the remark this answers is not in the document]
Comment (Scotchester) [reply packages/vue/src/components/BlockCardGrid/BlockCardGrid.stories.js:97]: Amazing that merely adding those three lines adds a whole story 🪄

[the remark this answers is not in the document]
Comment (stephiescastle) [reply packages/vue/src/components/BlockCardGrid/BlockCardGrid.stories.js:97]: Storybook has gotten much more streamlined!

**Pull request merged**